### PR TITLE
[2/3] FWI-4750 - CLI - CRUD PolicyMappings

### PR DESCRIPTION
**Pull request by vitorvezani**
This PR fixes #FWI-4750

Depends on https://github.com/FairwindsOps/insights-cli/pull/142

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### What's the goal of this PR?
- add `list policy-mappings` 
- add `push policy-mappings`
- add `download policy-mappings [--override]`
   - fails if there are files on directory, may use `--override` to override local files

### What alternative solution should we consider, if any?

[Internal Ticket FWI-4750](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-4750)

### CODE REVIEW

File: pkg/policymappings/policymappings.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+	upserts, deletes, err := comparePolicyMappings(pushDir, existingPolicyMappings)
+	if err != nil {
+		return fmt.Errorf("unable to compare and push policy-mapping to Insights: %w", err)
+	}
+
+	for _, policyMapping := range upserts {
+		logrus.Infof("upsert policy-mapping: %s", policyMapping.Name)
+		if !dryRun {
+			err = upsertPolicyMapping(org, insightsToken, host, policyMapping)
```
Comment (jdesouza) [inline pkg/policymappings/policymappings.go:53]: Wondering if we could have a way to do this in a batch?

File: pkg/policymappings/policymappings.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+				return fmt.Errorf("Error while upsert policy-mapping %s to Fairwinds Insights: %w", policyMapping.Name, err)
+			}
+		}
+	}
+
+	if deleteMissing {
+		for _, policyMappingForDelete := range deletes {
+			logrus.Infof("Deleting policy-mapping: %s", policyMappingForDelete.Name)
+			if !dryRun {
+				err = deletePolicyMapping(org, insightsToken, host, policyMappingForDelete)
```
Comment (jdesouza) [inline pkg/policymappings/policymappings.go:64]: Same here

File: pkg/policymappings/policymappings.go (hunk 1 above)
Comment (vitorvezani) [reply pkg/policymappings/policymappings.go:53]: There are no APIs for it just yet, frontend will use single API's, the payload is lightweight and orgs shouldn't have more than a dozen but could be an improvement

Comment (jdesouza) [reply pkg/policymappings/policymappings.go:53]: alright, thought they would have a lot of them

**Review by jdesouza — APPROVED** [review]

**Review by rbren — APPROVED** [review]

**Pull request merged**